.ko.yaml

```diff
@@ -1,2 +1,10 @@
+defaultBaseImage: cgr.dev/chainguard/static@sha256:67a1b00e0134e2b3a614c7198a26f7deed9d11b7acad4d52c79c0cfd47a2eae7
 baseImageOverrides:
+  # Use the combined base image for images that should include Windows support.
+  # NOTE: Make sure this list of images to use the combined base image is in sync with what's in tekton/publish.yaml's 'create-ko-yaml' Task.
```

Comment:
I wonder if we could also use this for `publish.yaml` and just replace any `latest` entry with resolved digest when running the release.